src/main/java/Duke.java

```diff
@@ -1,10 +1,73 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static Task[] tasks = new Task[100];
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke");
```

Comment:
This main function is quite long so if possible maybe it would be better to break it up into smaller methods for better readability!